### PR TITLE
i checked the homework answer

### PR DESCRIPTION
**Pull request by Amel1983**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website unique and attractive
```
Comment (Mohammedal-abdulhamid) [inline homework.md:31]: that is right . it make it attractive by changing the colours and layout of sections.

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-It send it to a friend
-
+It will dounlod in your local computer
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you send a file to review request
```
Comment (Mohammedal-abdulhamid) [inline homework.md:72]: It can also be to suggest a change to a repository

**Comment by seyedsharifi12** [comment]
Hi @Amel1983, I might be wrong but I think you have to fill in your details and the homework details in the pull_request_template.md file.

> # Your Details
> * Your Name:
> * Your City:
> * Your Slack Name:
> 
> # Homework Details
> * Module:
> * Week:

File: homework.md [hunk 3]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+A programming language used on the client-side and server-side
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
-
+edit a commit message
+remove the file frome git whihout removing it from github
```
Comment (seyedsharifi12) [inline homework.md:54]: Hi @Amel1983 there is typo here in the answer it says frome, should be from. 
You can also put something like GitHub solves the problem of version control and makes it easier to work collaboratively with people on projects from different parts of the work. 
Thank you.

**Pull request closed without merging**